honeycomb/honeycomb.go

```diff
@@ -80,10 +80,28 @@ type Exporter struct {
 
 // SpanEvent represents an event attached to a specific span.
 type SpanEvent struct {
-	Name          string `json:"name"`
-	TraceID       string `json:"trace.trace_id"`
-	ParentID      string `json:"trace.parent_id,omitempty"`
-	SpanEventType string `json:"meta.span_type"`
+	Name     string `json:"name"`
+	TraceID  string `json:"trace.trace_id"`
+	ParentID string `json:"trace.parent_id,omitempty"`
+	SpanType string `json:"meta.span_type"`
+}
+
+type SpanRefType int64
+
+const (
+	SpanRefType_CHILD_OF     SpanRefType = 0
+	SpanRefType_FOLLOWS_FROM SpanRefType = 1
+)
+
+// Link represents a link to a span that lives elsewhere.
+// TraceID and ParentID are used to identify the span with which the trace is associated
+type Link struct {
+	TraceID     string      `json:"trace.trace_id"`
+	ParentID    string      `json:"trace.parent_id,omitempty"`
```

Comment:
I understand why you want to make it a parentid as far as our data model, but... it also feels weird because this is a link to what would otherwise be the span's parent itself!

may want to add some comments to that specifically, that we are modeling it for now as a child of the event rather than a property of the event.

Reply:
Good idea. More comments coming ✅